okl/plugins/lowMach.okl

```diff
@@ -79,7 +80,8 @@
           const dfloat dtdz = vgeo[gid + p_TZID * p_Np];
           const dfloat JW   = vgeo[gid + p_JWID * p_Np];
 
-          const dfloat iRhoCpT = 1 / (RHOCP[id] * Q[id]);
+          //const dfloat iRhoCpT = 1 / (RHOCP[id] * Q[id]);
```

Comment:
Introduce a kernel parameter `p_ALPHAV`. If (p_ALPHAV == 1) compute else alpha = 0. This way we can use the same kernel for both usecases.